.github/workflows/ci_tests.yml

```diff
@@ -100,7 +106,8 @@ jobs:
       run: |
         python -m pip install -U pip
         pip install -U setuptools wheel
-        pip install -e .[dev]
+        pip install -U -r requirements.txt -r requirements_dev.txt
+        pip install -e .
```

Comment:
Why this change? Doesn't .[dev] isntall all standard requirements + dev requirements?

Reply:
I splittet it into two lines, because I want to install the requirements with the update (-U) option, which doesn't make sense for development (-e .) installation.